src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1114,69 +1123,251 @@ class Simplex_tree {
     return insert_simplex_raw(copy, filtration);
   }
 
-    /** \brief Insert a N-simplex and all his subfaces, from a N-simplex represented by a range of
+  /**
+   * @brief List of insertion strategies for @ref insert_simplex_and_subfaces, which takes a simplex \f$ \sigma \f$
+   * and a filtration value \f$ f \f$ as argument.
+   */
+  enum class Filtration_maintenance : std::uint8_t {
+    /**
+     * @brief Let \f$ f \f$ be the filtration value given as argument. Inserts the simplex \f$ \sigma \f$ as follows:
+     * - at value \f$ f \f$, if \f$ \sigma \f$ didn't existed yet,
+     * - if \f$ \sigma \f$ was already inserted, let \f$ f' \f$ be its filtration value. The new filtration value will
+     * be @ref FiltrationValue::unify_lifetimes (\f$ f \f$, \f$ f' \f$).
+     *
+     * Then, the filtration values of all faces of \f$ \sigma \f$ are pushed to the union of their old value and of
+     * the new value of \f$ \sigma \f$.
+     *
+     * So, this strategy can be interpreted as "insert the new simplices at the requested value \f$ f \f$ and insert
+     * already existing faces sooner in the filtration if necessary to maintain a valid filtration".
+     */
+    LOWER_EXISTING,
+    /**
+     * @brief Let \f$ f \f$ be the filtration value given as argument. Inserts the simplex \f$ \sigma \f$ as follows:
+     * - if \f$ \sigma \f$ was not inserted yet, then \f$ \sigma \f$ and all its faces, which are not already included
+     * in the complex, are inserted at either \f$ f \f$ or at the first possible filtration value when \f$ f \f$ is too
+     * low (to insure the validity of the filtration). This is computed with @ref FiltrationValue::intersect_lifetimes.
+     * - if \f$ \sigma \f$ existed already, then nothing is done.
+     *
+     * So, this strategy can be interpreted as "insert the new simplices as soon as the current filtration allows,
+     * with a lower bound at \f$ f \f$". If no lower bound is desired, \f$ f \f$ should be set to -inf or not specified
+     * at all.
+     */
```

Comment:
There is an issue in the documentation rendering, cf. this rendering (the same `\_form#272` at home):

<img width="1563" height="474" alt="Image" src="https://github.com/user-attachments/assets/89577b10-bb04-4e48-b0b6-ea80d3e394e3" />

It seems to come from the double quotes `So, this strategy can be interpreted as " ... \f$ f \f$ ..."` that are not on the same line. I think your sentence still makes sense without the double quotes, or you can modify with, for example: `So, this strategy can be interpreted as: Insert the ... \f$ f \f$ ...`